src/lib/cimb_translator/Common.cpp

```diff
@@ -46,18 +47,25 @@ namespace {
 
 namespace cimbar {
 
-cv::Mat load_img(string path, const string& image_dir)
+cv::Mat load_img(string path)
 {
-	if (image_dir != "")
-		return cv::imread(image_dir + "/" + path);
-
 	auto it = cimbar::bitmaps.find(path);
 	if (it == cimbar::bitmaps.end())
-		return cv::imread(path);
```

Comment:
We're losing the ability to load tiles locally here, but at this point it's only needed for tests.